onnxruntime/core/graph/graph_utils.cc

```diff
@@ -289,7 +289,7 @@ bool IsSupportedOptypeVersionAndDomain(const Node& node,
                                        const std::string& op_type,
                                        const std::initializer_list<ONNX_NAMESPACE::OperatorSetVersion>& versions,
                                        const std::string& domain) {
-  return (node.OpType() == op_type && !node.Op()->Deprecated() &&
+  return (node.OpType() == op_type && node.Op() && !node.Op()->Deprecated() &&
```

Comment:
would this be addressed by https://github.com/microsoft/onnxruntime/pull/4761?

Reply:
Not sure about it, will it? But it won't hurt to make sure node.Op() is not null before calling Deprecated(), this is where I am getting the NPE from.

Reply:
I found this code change is no longer applicable to the PR as there's other fix in place. So reverted, thx.

Reply:
Also tried the fix in https://github.com/microsoft/onnxruntime/pull/4761 , I can still repro.